src/data/mv_polynomial/comm_ring.lean

```diff
@@ -135,6 +138,32 @@ def hom_equiv : (mv_polynomial σ ℤ →+* S) ≃ (σ → S) :=
 
 end eval₂
 
+section degree_of
+
+lemma degree_of_sub_lt {x : σ} {f g : mv_polynomial σ R} {k : ℕ} (h : 0 < k)
+  (hf : ∀ (m : σ →₀ ℕ), m ∈ f.support → (k ≤ m x) → coeff m f = coeff m g)
+  (hg : ∀ (m : σ →₀ ℕ), m ∈ g.support → (k ≤ m x) → coeff m f = coeff m g) :
+  degree_of x (f - g) < k :=
+begin
+  rw degree_of_lt_iff,
```

Comment:
This creates two goals. The mathlib style is to surround the tactics on the lines that follow that work on each goal in `{}`, to make it clear which goal is currently being worked. on.

Reply:
Although in this particular case:
```suggestion
  rw degree_of_lt_iff h,
```